src/.env.sample

```diff
@@ -1 +1 @@
-BACKEND_URL=
\ No newline at end of file
+REACT_APP_BACKEND_URL=
```

Comment:
No need for a unique name. This variable is unique

Reply:
Is this variable variable in process.env. Try change this name and make console.log. You will surprised. Maybe something is missing or donetv is required?

Reply:
When you add this variable to your env, it will be accessed in process.env

Reply:
[screenshots.zip](https://github.com/denyskuz/slimmom/files/10271891/screenshots.zip)
Here some screenshots. If I say it is not working, it mean I try it and it fail. I mention that early. Why we spent time on so stupid thing. It is so hard open branch and test it? And finally you can read react [docs](https://create-react-app.dev/docs/adding-custom-environment-variables/). If you have any suggestions I will glad to see them.

Reply:
ok